pypesto/visualize/sampling.py

```diff
@@ -57,9 +59,24 @@ def sampling_fval_trace(
     kwargs = {'edgecolor': "w",  # for edge color
               'linewidth': 0.3,
               's': 10}
+    if full_trace:
+        kwargs['hue'] = "converged"
+        kwargs['palette'] = ["#868686", "#477ccd"]
```

Comment:
is this necessary? in general not in favor of manually specifying colors ...

Reply:
i specify the color just to have same color as for the converged trace. If I do not define the color name would be such that warm up is "blue" and converged is "orange". I wanted to keep it consistent and I also thought that light grey would be a good color choice for the warm up.

Reply:
ok, I think good for the moment then, to be maybe changed when someone uses it and wants sth else.

Reply:
should I implement this now for this PR?

Reply:
anything fine I guess